src/api/middleware/participantsMiddleware.ts

```diff
@@ -20,7 +20,13 @@ const hasParticipantAccess = async (req: ParticipantRequest, res: Response, next
   }
 
   const currentUserEmail = req.auth?.payload?.email as string;
-  if (!(await isUserBelongsToParticipant(currentUserEmail, participantId, traceId))) {
+  const currentUserIsUid2Support = await userHasUid2SupportRole(currentUserEmail);
+
+  const userHasAccessToParticipant =
+    currentUserIsUid2Support ||
+    (await isUserBelongsToParticipant(currentUserEmail, participantId, traceId));
+
+  if (!userHasAccessToParticipant) {
```

Comment:
nit: based on naming conventions and the pattern of putting the verb first (is/can/has), maybe makes more sense to be `hasUserUid2SupportRole` or `hasUserAccessToParticipant` but up to you

Reply:
good suggestion! I've renamed this and a few other variables/methods.